packages/fx-core/src/plugins/resource/bot/v2/mapping.ts

```diff
@@ -59,6 +67,14 @@ export const moduleMap: { [key: string]: string } = {
   [ServiceType.Functions]: BicepModules.Functions,
 };
 
+export function getPlatformRuntime(platform: Platform): Runtime {
+  const runtime = PlatformRuntimeMap.get(platform);
+  if (runtime) {
+    return runtime;
+  }
+  throw new Error(invalidInputMsg);
```

Comment:
NIT: We better to clarify what input is invalid in the error message. User can't figure out what happened when getting such error `BT.UnhandledError: Invalid bot input`.

Reply:
This error message will never be shown to the users unless our code has bugs, and that's why I didn't localize this error message. But you are right at some point, I will find ways to make these error message better in the future.